### ISSUE TITLE
feature bank (LMDB)

### ISSUE THREAD

**Issue by yan-ctrl** [TencentYoutuResearch/ActionDetection-LSTC#1]
I like your paper very much. Can you tell me how to get the feature bank (LMDB)? I want to apply your work to my own data set. Do you run the script extract_feature.py directly or extract it like LFB?

**Comment by lyxok1** [comment]
@yan-ctrl Yes, the LMDB file of feature bank is obtained through `scripts/extract_feature.py`. To do this, you have to train a pure action detection network without feature bank first and load it during feature extraction, this may involves some code modification.

**Comment by yan-ctrl** [comment]
OK, thank you very much for your reply. I need to try it.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "TencentYoutuResearch/ActionDetection-LSTC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月29日(星期一) 下午2:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [TencentYoutuResearch/ActionDetection-LSTC] feature bank (LMDB) (Issue #1)





 
@yan-ctrl Yes, the LMDB file of feature bank is obtained through scripts/extract_feature.py. To do this, you have to train a pure action detection network without feature bank first and load it during feature extraction, this may involves some code modification.

**Comment by yan-ctrl** [comment]
> scripts/extract_feature.py

I'd like to ask again, I read the 99 lines of your scripts/extract_feature.py: scripts/extract _ feature.py. It seems that you have defined the write function to store the feature library LMDB mentioned in your paper, but I found that the LMDB package is missing, and 12 lines: import lmdb has an error. Could you tell me how to do it?

**Comment by lyxok1** [comment]
Till now maybe you should manually install lmdb package for python `pip install lmdb`, I will also update the setup script later.

**Comment by yan-ctrl** [comment]
> Till now maybe you should manually install lmdb package for python `pip install lmdb`, I will also update the setup script later.

Thank you, I installed it. I thought it was the module in your library that needed to be compiled.

**Comment by yan-ctrl** [comment]
> Till now maybe you should manually install lmdb package for python `pip install lmdb`, I will also update the setup script later.

Do you have a link to Baidu Cloud of feature bank(LMDB)? Now I'm having trouble accessing Google Cloud Disk, which was possible before.

**Comment by lyxok1** [comment]
Currently we only have LMDB files on Google Drive, the original LMDB file on server have been removed, I will try to download it from Google Drive and upload it once again to Baidu Cloud if necessary.

**Comment by lyxok1** [comment]
Currently, the LMDB file is too big (> 4GB) to be uploaded to Baidu Cloud

**Comment by yan-ctrl** [comment]
OK, now the access to the external network is restricted. I'll find the omnipotent Taobao.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "TencentYoutuResearch/ActionDetection-LSTC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月31日(星期三) 晚上8:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [TencentYoutuResearch/ActionDetection-LSTC] feature bank (LMDB) (Issue #1)





 
Currently, the LMDB file is too big (&gt; 4GB) to be uploaded to Baidu Cloud

**Comment by yan-ctrl** [comment]
Ok, thank you. I'll go to Taobao to see if there are any downloads.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "TencentYoutuResearch/ActionDetection-LSTC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月31日(星期三) 晚上8:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [TencentYoutuResearch/ActionDetection-LSTC] feature bank (LMDB) (Issue #1)





 
Currently we only have LMDB files on Google Drive, the original LMDB file on server have been removed, I will try to download it from Google Drive and upload it once again to Baidu Cloud if necessary.
